extra/interface/src/main/java/org/spongepowered/configurate/interfaces/InterfaceDefaultOptions.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Configurate
+ * Copyright (C) zml and Configurate contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.spongepowered.configurate.interfaces;
+
+import java.util.function.UnaryOperator;
+import org.spongepowered.configurate.ConfigurationOptions;
+import org.spongepowered.configurate.serialize.TypeSerializerCollection;
+
+/**
+ * This class has the default {@link ConfigurationOptions}
+ * with {@link InterfaceTypeSerializer} added to the serializers.
+ *
+ * @since 4.2.0
+ */
+public final class InterfaceDefaultOptions {
+
+    private static final ConfigurationOptions DEFAULTS =
+        ConfigurationOptions.defaults()
+            .serializers(
+                TypeSerializerCollection.builder()
+                    .registerAnnotated(InterfaceTypeSerializer::applicable, InterfaceTypeSerializer.INSTANCE)
+                    .registerAnnotatedObjects(InterfaceConstraints.buildObjectMapperWithConstraints())
+                    .registerAll(TypeSerializerCollection.defaults())
+                    .build()
+            );
+
+    private InterfaceDefaultOptions() {
+    }
+
+    /**
+     * The default ConfigurationOptions with {@link InterfaceTypeSerializer} added to the serializers.
+     *
+     * @return the default ConfigurationOptions with {@link InterfaceTypeSerializer} added to the serializers.
+     * @since 4.2.0
+     */
+    public static ConfigurationOptions get() {
```

Comment:
It's better to expose just the TSC rather than a whole `ConfigurationOptions` instance. YAML users who use their own options instance already have issues as-is cuz snakeyaml has its own serialization logic.